### PR TITLE
feat: add cache max size setting to ResourceService

### PR DESCRIPTION
**Pull request by artsiomkorzun**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,7 +77,8 @@ Priority order:
 | storage.maxUploadedFileSize                   |     536870912      |    No    |Maximum size in bytes of uploaded file. If a size of uploaded file exceeds the limit the server returns HTTP code 413
 | encryption.secret                             |         -          |    No    |Secret is used for AES encryption of a prefix to the bucket blob storage. The value should be random generated string.
 | encryption.key                                |         -          |    No    |Key is used for AES encryption of a prefix to the bucket blob storage. The value should be random generated string.
-| resources.maxSize                             |      1048576       |    No    |Max allowed size in bytes for a resource.
+| resources.maxSize                             |      67108864      |    No    |Max allowed size in bytes for a resource.
+| resources.cacheMaxSize                        |      1048576       |    No    |Max size in bytes for a resource to cache in Redis.
```
Comment (astsiapanay) [inline README.md]: maxCachedResourceSize - max size in bytes of a resource to cache in Redis?
cacheMaxSize has a different meaning to me

File: storage/src/main/java/com/epam/aidial/core/storage/service/ResourceService.java [hunk 2]
```diff
@@ -90,6 +90,7 @@ public class ResourceService implements AutoCloseable {
     private final ResourceTopic topic;
     @Getter
     private final int maxSize;
+    private final int cacheMaxSize;
```
Comment (astsiapanay) [inline storage/src/main/java/com/epam/aidial/core/storage/service/ResourceService.java]: please add comments

File: README.md (hunk 1 above)
Comment (astsiapanay) [reply README.md]: or maxCachedSize

File: storage/src/main/java/com/epam/aidial/core/storage/service/ResourceService.java (hunk 2 above)
Comment (artsiomkorzun) [reply storage/src/main/java/com/epam/aidial/core/storage/service/ResourceService.java]: Already commented in Settings class.

File: README.md (hunk 1 above)
Comment (artsiomkorzun) [reply README.md]: renamed to maxSizeToCache

**Review by astsiapanay — APPROVED** [review]

**Pull request merged**